cypress/e2e/vendingMachine.cy.js

```diff
@@ -0,0 +1,248 @@
+import { ERROR_MESSAGE } from '../../src/js/constants/errorMessage';
+import { SELECTOR } from '../../src/js/constants/selector';
+
+const PRODUCT_INDEX = {
+  NAME: 0,
+  PRICE: 1,
+  QUANTITY: 2,
+};
+
+beforeEach(() => {
+  cy.visit('index.html');
+});
+
+describe('상품관리 탭을 테스트한다.', () => {
+  context('상품을 추가할 때', () => {
+    it('최초 상품 목록은 비워진 상태이다.', () => {
+      cy.get('#product-inventory-container').children().should('have.length', 0);
+    });
+
+    it('상품명은 공백이 불가능하다.', () => {
+      const alertStub = cy.stub();
+      cy.on('window:alert', alertStub);
+
+      cy.typeProductPrice(1000);
+      cy.typeProductQuantity(5);
+
+      cy.clickProductAddButton().then(() => {
+        expect(alertStub.getCall(0)).to.be.calledWith(ERROR_MESSAGE.EMPTY_INPUT);
+      });
```

Comment:
```suggestion
      const stub = getAlertStub();
      
      cy.typeProductPrice(1000);
      cy.typeProductQuantity(5);

      cy.clickProductAddButton().then(() => {
        stub.calledWith(ERROR_MESSAGE.EMPTY_INPUT);
      });
```
이런 인터페이스로 함수가 작성되면 편할 것 같아요 👀

Reply:
훨씬 깔끔하고 편하니 좋네요!
수정해보겠습니다!

Reply:
요것은 혹시 시도했는데 잘 안되셨을까요? 👀